core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -70,6 +70,11 @@ angular.module('oppia').factory('ResponsesService', [
     var _confirmedUnclassifiedAnswers = null;
     var _answerChoices = null;
 
+    // A boolean flag to check whether we need to update and save the
+    // answer groups when the answer choices are modified in the case of
+    // MultipleChoiceInput interaction.
+    var _multiChoiceInputAnswerGroupRequiresUpdation = false;
```

Comment:
Updation --> Update

Also, I'm not sure about this approach. Is it possible to do something similar to item selection and not keep a flag like this? The problem with the flag is that it's not really clear what the flag is signalling -- it would be better to just do the update directly rather than storing state like this, if possible.

Reply:
done